tests/datadriven/__init__.py

```diff
@@ -194,6 +200,24 @@ def assertRaisesWith(self, exceptionType):
                 "exception of type {} not raised".format(
                     exceptionType.__name__))
 
+    def assertAlmostEqual(self, a, b, ndigits=7):
+        """
+        Assert that a and b are equal within ndigits digits
+        """
+        if round(a-b, ndigits) != 0:
```

Comment:
@pgrosu I can assure you that negative numbers do indeed exist :)

All of the above being said, I'm going to defer to those of you who are already using `assertAlmostEqual` in the test code, as to whether this significant digit or the previously proposed decimal places implementation is more useful in practice...